lib/Drawing/Color.vala

```diff
@@ -54,311 +54,331 @@ namespace Granite.Drawing {
          * @param A the value of the alpha channel as a double
          */
         public Color (double R, double G, double B, double A) {
-        
+
             this.R = R;
             this.G = G;
             this.B = B;
             this.A = A;
         }
-        
+
         /**
          * Constructs a new {@link Granite.Drawing.Color} from a {@link Gdk.Color}.
          *
          * @param color the {@link Gdk.Color}
          */
         public Color.from_gdk (Gdk.Color color) {
-        
+
             R = color.red / (double) uint16.MAX;
             G = color.green / (double) uint16.MAX;
             B = color.blue / (double) uint16.MAX;
             A = 1.0;
         }
-        
+
+        /**
+         * Constructs a new {@link Granite.Drawing.Color} from a {@link Gdk.RGBA}.
+         *
+         * @param color the {@link Gdk.RGBA}
+         */
+        public Color.from_rgba (Gdk.RGBA color) {
+            set_from_rgba (color);
+        }
+
+        /**
+         * Constructs a new {@link Granite.Drawing.Color} from a string.
+         *
+         * @param color the string specifying the color
+         */
+        public Color.from_string (string color) {
```

Comment:
Mention the `Granite.Color.to_string` here since current description doesn't tell anything about the format.

`For more details on formatting and how this function works see {@link Granite.Color.to_string}`